5.PropTypes and DefaultProps.md

```diff
@@ -0,0 +1,40 @@
+#Prop Types And Default Props
+
+##Prop Types
+PropTypes provides **typechecking ability** to react components.
+Let's take a look at some codes.
+
+```
```

Comment:
There is the syntax, but not sure if there's any difference.
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1988